protostar/cli/protostar_arg_type.py

```diff
@@ -19,17 +21,22 @@
 def map_protostar_type_name_to_parser(
     argument_type: ProtostarArgTypeName,
 ) -> Callable[[str], Any]:
+    parser = None
     if argument_type == "felt":
-        return parse_felt_arg_type
-    if argument_type == "fee":
-        return parse_fee_arg_type
-    if argument_type == "address":
-        return parse_hex_or_decimal
-    if argument_type == "class_hash":
-        return parse_hex_or_decimal
-    if argument_type == "wei":
-        return parse_wei_arg_type
-    return map_type_name_to_parser(argument_type)
+        parser = parse_felt_arg_type
+    elif argument_type == "fee":
+        parser = parse_fee_arg_type
+    elif argument_type == "address":
+        parser = parse_hex_or_decimal
+    elif argument_type == "class_hash":
+        parser = parse_hex_or_decimal
+    elif argument_type == "wei":
+        parser = parse_wei_arg_type
+    elif argument_type == "input":
+        parser = parse_input_arg_type
+    else:
+        parser = map_type_name_to_parser(argument_type)
+    return parser
```

Comment:
Had to do a refactor this way due to a lint error - too many returns.

Reply:
Ignore  error, it's dumb

Reply:
I agree it is, but if we want to ignore it, I think we should remove the rules responsible for it from `pylint.rc`: `max-returns=6`

Reply:
> I agree it is, but if we want to ignore it, I think we should remove the rules responsible for it from `pylint.rc`: `max-returns=6`

Agree

Reply:
nitpick: @karol-bisztyga could you please move max returns removal to separate PR? It's out of scope of this PR actually.

Reply:
#1153